helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml

```diff
@@ -3,15 +3,22 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}
+  name: {{ $fullName }}-ingress
```

Comment:
It needs a new name, so that it's separate from the old ones, I think it can be anything.  I used `-ingress` for the backend ones so using that again here.